substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/NativeImageOptions.java

```diff
@@ -50,15 +50,20 @@
 
     public static final int DEFAULT_MAX_ANALYSIS_SCALING = 16;
 
-    @Option(help = "Comma separated list of CPU features that will be used for image generation. " +
-                    "The specific options available are platform dependent. " +
+    @Option(help = "Comma separated list of CPU features that will be enabled while building the " +
+                    "target executable, irrespective of whether they are supported by the hosted " +
+                    "environment. Note that enabling features not present within the target environment " +
+                    "may result in application crashes. The specific options available are target " +
+                    "platform dependent. " +
                     "For AMD64, SSE and SSE2 are enabled by default. Available features are: " +
                     "CX8, CMOV, FXSR, HT, MMX, AMD_3DNOW_PREFETCH, SSE3, SSSE3, SSE4A, SSE4_1, " +
-                    "SSE4_2, POPCNT, LZCNT, TSC, TSCINV, AVX, AVX2, AES, ERMS, CLMUL, BMI1, " +
+                    "SSE4_2, POPCNT, LZCNT, TSC, TSCINV, TSCINV_BIT, AVX, AVX2, AES, ERMS, CLMUL, BMI1, " +
```

Comment:
It clearly does not scale well to manually list all the available flags for all architectures in this help message...

Reply:
I agree. I would suggest adding support for a sentinel feature named `-HELP` that will list all other features if specified.

Reply:
I added a native-image driver option `--list-cpu-features` which lists the features and then exits. This option is listed under `--help-extra`